legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/protocols/pure/vX_X_X/extension/extension_serviceStore.pure

```diff
@@ -17,6 +17,8 @@ import meta::pure::mapping::*;
 import meta::external::store::service::metamodel::*;
 import meta::external::store::service::metamodel::mapping::*;
 import meta::external::store::service::metamodel::runtime::*;
+import meta::pure::functions::collection::*;
+import meta::protocols::pure::vX_X_X::metamodel::executionPlan::store::service::SingleAuthScheme::*;
```

Comment:
lets not include any vX_X_X package in import and have such package directly where its needed. Also lets move pure above